neothesia/src/scene/menu_scene/iced_menu/settings.rs

```diff
@@ -241,11 +244,27 @@ fn output_group<'a>(data: &'a Data, ctx: &Context) -> Element<'a, Event> {
             .suffix(counter(ctx.config.audio_gain(), Event::AudioGain))
     });
 
+
+    let separate_channels_toggler = toggler(ctx.config.separate_channels())
+        .on_toggle(Event::SeparateChannels)
+        .style(theme::toggler);
+
+    let separate_channels_settings = mouse_area(
+        ActionRow::new()
+            .title("Separate Channels")
+            .subtitle("Assign different MIDI channel to each track")
+            .suffix(separate_channels_toggler),
+    )
+        .on_press(Event::SeparateChannels(
+            !ctx.config.separate_channels(),
+        ));
+
```

Comment:
This only makes sense for MIDI output, right?
If so, let's make it show up only for MIDI outputs

```suggestion
    let is_midi = matches!(data.selected_output, Some(OutputDescriptor::MidiOut(_)));
    let separate_channels_settings = is_midi.then_some(separate_channels_settings);
```

And futher down:
```diff
- .push(separate_channels_settings)
+ .push_maybe(separate_channels_settings)
```

Reply:
Done